### PR TITLE
feat(VPCEP): add datasource to query VPC endpoint service connections

### PR DESCRIPTION
**Pull request by MrLiangSir**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

make testacc TEST='./huaweicloud/services/acceptance/vpcep' TESTARGS='-run TestAccDatasourceVPCEPServiceConnections_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpcep -v -run TestAccDatasourceVPCEPServiceConnections_basic -timeout 360m -parallel 4
=== RUN   TestAccDatasourceVPCEPServiceConnections_basic
=== PAUSE TestAccDatasourceVPCEPServiceConnections_basic
=== CONT  TestAccDatasourceVPCEPServiceConnections_basic
--- PASS: TestAccDatasourceVPCEPServiceConnections_basic (240.72s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpcep     240.779s

### CODE REVIEW

File: docs/data-sources/vpcep_service_connections.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* `region` - (Optional, String) The region in which to obtain the VPC endpoint service. If omitted, the
+  provider-level region will be used.
+
+* `service_id` - (Required, String) Specifies the ID of VPC endpoint service.
+
+* `endpoint_id` - (Optional, String) Specifies the ID of VPC endpoint which has connected to
+  VPC endpoint service.
+
+* `status` - (Optional, String) Specifies the connection status of the VPC endpoint.
+  The value can be **pendingAcceptance** , **accepted**, **rejected** and **failed**.
```
Comment (deer-hang) [inline docs/data-sources/vpcep_service_connections.md]:
```suggestion
  The value can be **pendingAcceptance**, **accepted**, **rejected** and **failed**.
```

File: huaweicloud/services/acceptance/vpcep/data_source_huaweicloud_vpcep_service_connections_test.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		},
+	})
+}
+
+func testAccDatasourceVpcepServiceConnections_basic(name string) string {
+	return fmt.Sprintf(`
+%s
+
+data "huaweicloud_vpcep_service_connections" "test" {
+  depends_on = [huaweicloud_vpcep_endpoint.test]
```
Comment (deer-hang) [inline huaweicloud/services/acceptance/vpcep/data_source_huaweicloud_vpcep_service_connections_test.go]: There is no need to add this line of code.

**Comment by MrLiangSir** [comment]
make testacc TEST='./huaweicloud/services/acceptance/vpcep' TESTARGS='-run TestAccDatasourceVPCEPServiceConnections_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpcep -v -run TestAccDatasourceVPCEPServiceConnections_basic -timeout 360m -parallel 4
=== RUN   TestAccDatasourceVPCEPServiceConnections_basic
=== PAUSE TestAccDatasourceVPCEPServiceConnections_basic
=== CONT  TestAccDatasourceVPCEPServiceConnections_basic
--- PASS: TestAccDatasourceVPCEPServiceConnections_basic (239.21s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpcep     239.292s

**Pull request merged**